specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2462,13 +2497,21 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+			"UserAssigned"
           ],
           "x-ms-enum": {
             "name": "IdentityType",
             "modelAsString": false
           }
-        }
+        },
+		"userAssignedIdentities":{
+		  "type": "string",
+		  "description": "Dictionary to map userAssignedIdentityID strings to userAssignedIdentity properties",
+		  "additionalProperties": {
+		    "type": "#/definitions/UserAssignedIdentityType"
+		  }
+		}
```

Comment:
SDK generation will fail with this swagger change.

> ERROR: Schema violation: Data does not match any schemas from 'anyOf'
>     - file:///C:/code/swagger/specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json:2526:10 ($.definitions.Identity.properties.userAssignedIdentities.additionalProperties)
> FATAL: swagger-document/individual/schema-validator - FAILED
> FATAL: Error: [OperationAbortedException] Error occurred. Exiting.
> Process() cancelled due to exception : [OperationAbortedException] Error occurred. Exiting.

Reply:
Acked.

Checking. Sorry for the delays on this.

Reply:
I should have used "$ref" instead of "type". Will make that change soon.